src/hiposfer/kamal/network/generators.clj

```diff
@@ -47,4 +50,5 @@
     (for [id ids]
       {:way/id id
        :way/name (gen/generate namer)
-       :way/nodes (random-sample 0.2 ids)})))
+       :way/nodes (for [i (random-sample 0.4 ids)]
+                    [:node/id i])})))
```

Comment:
@mehdisadeghi this is where my generator was completely broken. It was creating a data structure that is not consistent with the schema of Datascript, thus it associated a single node with a way, instead of many nodes to a way.

The incosistency that this caused was 10 times worse than the one that we currently have, which is why I was able to pinpoint it 😅